### PR TITLE
Modify the price_volume_diff from datetime so it can have data for all points

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)
Previously: 
image

Now: 
image

### CODE REVIEW

File: lib/sanbase/tech_indicators/price_volume_difference.ex [hunk 1]
```diff
@@ -42,6 +42,16 @@ defmodule Sanbase.TechIndicators.PriceVolumeDifference do
       ) do
     url = "#{tech_indicators_url()}/indicator/pricevolumediff/ma"
 
+    # The calculation needs to start from before the `from_datetime` so the
```
Comment (kalkov) [inline lib/sanbase/tech_indicators/price_volume_difference.ex]: You can add a `HACK` annotation :smiley_cat:

File: test/sanbase/internal_services/tech_indicators_test.exs [hunk 2]
```diff
@@ -27,7 +27,7 @@ defmodule Sanbase.TechIndicatorsTest do
...
+           "[#{Sanbase.TechIndicatorsTestResponse.price_volume_diff_prepend_response()},{\"price_volume_diff\": 0.0, \"price_change\": 0.04862261825993345, \"volume_change\": 0.030695260272520467, \"timestamp\": 1516406400}, {\"price_volume_diff\": -0.014954423076923185, \"price_change\": 0.04862261825993345, \"volume_change\": 0.030695260272520467, \"timestamp\": 1516492800}, {\"price_volume_diff\": -0.02373337292856359, \"price_change\": 0.04862261825993345, \"volume_change\": 0.030695260272520467, \"timestamp\": 1516579200}, {\"price_volume_diff\": -0.030529013702074614, \"price_change\": 0.04862261825993345, \"volume_change\": 0.030695260272520467, \"timestamp\": 1516665600}, {\"price_volume_diff\": -0.0239400614928722, \"price_change\": 0.04862261825993345, \"volume_change\": 0.030695260272520467, \"timestamp\": 1516752000}]",
```
Comment (kalkov) [inline test/sanbase/internal_services/tech_indicators_test.exs]: Can this be a multi-line string? It will be easier to read.

Comment (IvanIvanoff) [reply test/sanbase/internal_services/tech_indicators_test.exs]: Fixed

File: lib/sanbase/tech_indicators/price_volume_difference.ex (hunk 1 above)
Comment (IvanIvanoff) [reply lib/sanbase/tech_indicators/price_volume_difference.ex]: Added a line that this is a workaround for the not that good tech_indicator's api

**Pull request merged**